kit/brain_board/index.md

```diff
@@ -24,12 +24,12 @@ The LEDs on the HAT display the current status of the robot and can be used to h
 
```

Comment:
We currently don't mention on the brain board page that it needs to be L2, nor do we mention how to power it. Do we think this is worth doing, or should we leave it just on the power board page?

Copying over the warning feels like it might be enough?

Reply:
> We currently don't mention on the brain board page that it needs to be L2, nor do we mention how to power it. Do we think this is worth doing, or should we leave it just on the power board page?
> 
> Copying over the warning feels like it might be enough?

This was discussed on the PR which updated that page (#355) and we concluded that it should be mentioned on the kit assembly page, as that's where this sort of thing has gone in the past. That page does mention this requirement and the fact that it needs a CamCon based cable (#369).

Reply:
I think it's worth saying that it's powered from 12 volts to avoid confusion.

Reply:
[79f1ece](https://github.com/srobo/docs/pull/375/commits/79f1ece33799ec29e40577ab284230cd65f0a1e4)